classes/User.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * User Class
+ *
+ * @package TheyWorkForYou
+ */
+
+namespace MySociety\TheyWorkForYou;
+
+/**
+ * User
+ */
+
+class User {
+    public function getMP($cons_type, $mp_house) {
```

Comment:
Is getMP/mp_house the right name, if it's returning other things and mp_house might be Scotland? :)